openmp/runtime/src/kmp.h

```diff
@@ -1181,7 +1181,11 @@ extern void __kmp_init_target_task();
 #define KMP_MIN_STKSIZE ((size_t)(32 * 1024))
 #endif
 
+#if KMP_OS_AIX && KMP_ARCH_PPC
```

Comment:
Do we really need `KMP_ARCH_PPC` here?

Reply:
`KMP_ARCH_PPC` is for 32-bit PPC. It is needed here because this definition is only for 32-bit.

Reply:
Got it. Thanks.

Reply:
I had similar doubts when I saw the first AIX patch months ago. 😄